config.cson

```diff
@@ -1751,6 +1751,7 @@ fileIcons:
 		match: [
 			[".css", "medium-blue", "css"]
 			[".less", "dark-blue", "less"]
+			[".tss", "medium-blue", "tss"]
```

Comment:
```suggestion
			[".tss", "dark-red", "tss"]
```

**Rationale:** [Appcelerator's logo](https://en.wikipedia.org/wiki/File:Appcelerator_logo.svg) uses red, and `.tss` files should ideally have a unique colour to distinguish them from CSS and Less files.

Reply:
I like that!